pkg/libartifact/store/store.go

```diff
@@ -29,6 +29,7 @@ import (
 
 var (
 	ErrEmptyArtifactName = errors.New("artifact name cannot be empty")
+	SchemaVersion        = 2
```

Comment:
This should be put into a const block, and maybe name it `ManifestSchemaVersion` so we now want schema it is talking about.

Reply:
I’m not truly convinced this is worth making a constant. For `specV1.Manifest.Version`, the only valid value is `2`, and that’s unlikely to change (a change would certainly mean a new MIME type and _probably_ a new Go type).

It’s not a parameter that can be tuned without changing the type references.

The _clean_ location where this constant should exist is somewhere in the `opencontainers/image-spec` repo.

All that said, now that the code to declare a constant exists, I’m not going to ask that it should be replaced by a hard-coded `2`.